src/Containers/MiddleCanvas/DroppableElement.tsx

```diff
@@ -0,0 +1,78 @@
+import React from 'react';
+import styled from 'styled-components';
+import {
+    Droppable, 
+    DroppableProvided, 
+    DroppableStateSnapshot 
+} from 'react-beautiful-dnd';
+import { StyledIcon } from 'styled-icons/types';
+import { DroppableContainerContents } from './DroppableContainerContents';
+import { MainInterface, ResponsiveInterface } from '../../Utils/BaseStyles';
+
+export interface DroppableElementProps extends MainInterface, ResponsiveInterface, React.HTMLAttributes<HTMLDivElement> {
+    droppableLabels: string[][] | StyledIcon [][],
+    styling: string, 
+    isColumn: boolean,
+    isPreview?: boolean
+};
+
+interface DroppableContainerProps {
+    isDragging?: boolean,
+    isColumn?: boolean,
+    isPreview?: boolean
+};
+
+const FIRST_LABEL = 0;
+const DESIGN_BACKGROUND_COLOR = '#ffffff';
+const PREVIEW_BACKGROUND_COLOR = '#f5f5f5';
+const DESIGN_COLOR = '#b7b7b7';
+const PREVIEW_COLOR = 'black';
+
+export const DroppableElement: React.FC<DroppableElementProps> = ({
+    droppableLabels,
+    styling,
+    isColumn,
+    isPreview,
+    ...props
+}): React.ReactElement => {
+    return (
+        <Wrapper {...props}>
+            {Object.values(droppableLabels).map(droppableLabel => (
```

Comment:
no need

Reply:
Try to avoid writing logic instead the tree.

<Wrapper>
{getDraggableObject()}
</Wrapper>